src/lib/components/DisclaimerModal.svelte

```diff
@@ -37,8 +37,9 @@
 					class:text-gray-800={$page.data.loginEnabled}
 					class:hover:bg-slate-100={$page.data.loginEnabled}
 					on:click={(e) => {
+						e.preventDefault();
+						e.stopPropagation();
```

Comment:
```suggestion
					on:click|preventDefault|stopPropagation={(e) => {
```

haven't tested it. But is there a reason not to use [svelte event modifiers](https://svelte.dev/docs/element-directives#on-eventname)?

Reply:
Reason was that I forgot about them haha, thanks for catching it, I'll fix it everywhere.